### ISSUE TITLE
SDSS query is broken

### ISSUE THREAD

**Issue by aileisun** [astropy/astroquery#685]
It was working until about three days ago, but now was broken and returns an error message. It's probably because of SDSS changing their new URL to:

http://skyserver.sdss.org/SkyserverWS/dr12/SearchTools/SqlSearch

```
In [19]:  xid = SDSS.query_region(pos, spectro=True)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-19-077e4697f30b> in <module>()
----> 1 xid = SDSS.query_region(pos, spectro=True)

/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/astroquery/utils/class_or_instance.pyc in f(*args, **kwds)
     23         def f(*args, **kwds):
     24             if obj is not None:
---> 25                 return self.fn(obj, *args, **kwds)
     26             else:
     27                 return self.fn(cls, *args, **kwds)

/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/astroquery/utils/process_asyncs.pyc in newmethod(self, *args, **kwargs)
     27             if kwargs.get('get_query_payload') or kwargs.get('field_help'):
     28                 return response
---> 29             result = self._parse_result(response, verbose=verbose)
     30             self.table = result
     31             return result

/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/astroquery/sdss/core.pyc in _parse_result(self, response, verbose)
    833                                           names=True, dtype=None,
    834                                           delimiter=',', skip_header=1,
--> 835                                           comments='#'))
    836 
    837         if len(arr) == 0:

/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy/lib/npyio.pyc in genfromtxt(fname, dtype, comments, delimiter, skip_header, skip_footer, converters, missing_values, filling_values, usecols, names, excludelist, deletechars, replace_space, autostrip, case_sensitive, defaultfmt, unpack, usemask, loose, invalid_raise, max_rows)
   1767             # Raise an exception ?
   1768             if invalid_raise:
-> 1769                 raise ValueError(errmsg)
   1770             # Issue a warning ?
   1771             else:

ValueError: Some errors were detected !
    Line #60 (got 4 columns instead of 1)
    Line #123 (got 6 columns instead of 1)
    Line #124 (got 7 columns instead of 1)
    Line #125 (got 4 columns instead of 1)
    Line #126 (got 4 columns instead of 1)
    Line #127 (got 4 columns instead of 1)
    Line #129 (got 2 columns instead of 1)
```

**Comment by bsipocz** [comment]
Thanks @aileisun, I'll look into this over the weekend.

**Comment by rmutel** [comment]
I can confirm the exact same error, and timing (it worked until sometime Thursday).

**Comment by bsipocz** [comment]
@aileisun @rmutel - Fix is in #690, I'm planning to merge it tomorrow to give a bit of time for others to comment on it.

**Comment by bsipocz** [comment]
@aileisun @rmutel - Updating your astroquery to the latest dev version will have the fix. Alternatively you can wait for the next release that we plan to do soon.
